client/src/App.js

```diff
@@ -3,15 +3,16 @@ import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route } from "react-router-dom";
 
 import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
+//import LandingPage from "./pages/Landing";
+import SignUpPage from "./pages/SignUp";
 
 import "./App.css";
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Route path="/" component={LandingPage} />
+        <Route path="/" component={SignUpPage} />
```

Comment:
we should probably have this on a route like `/signup`

Reply:
Sure. Done.